### ISSUE TITLE
Bump into the new versions?

### ISSUE THREAD

**Issue by iamflorencejay** [AzuraCast/icecast-kh-ac#15]
I think Icecast-kh is unmaintained for quite a long time now, so is it possible to bump it into the latest version of icecast?

**Comment by ItsMitchh** [comment]
We can't bump to IceCast because they're missing some things that the KH branch has.

**Comment by iamflorencejay** [comment]
> We can't bump to IceCast because they're missing some things that the KH branch has.

Can I ask if it is possible to clone the icecast and icecast-kh, and compare & error correct?

**Comment by ItsMitchh** [comment]
Possible? Yes. 
Do we have the time to dedicate to it? Perhaps? Can't say for certain, this'll involve a lot of work, namely testing, and debugging with code (at least myself) isn't knowledgable with.

**Comment by iamflorencejay** [comment]
> Possible? Yes. Do we have the time to dedicate to it? Perhaps? Can't say for certain, this'll involve a lot of work, namely testing, and debugging with code (at least myself) isn't knowledgable with.

yeah, that's sucks. I wanna implement http-headers on icecast so much.

**Comment by ItsMitchh** [comment]
I'll raise this up and see what we can do as a long term solution.

**Comment by iamflorencejay** [comment]
> I'll raise this up and see what we can do as a long term solution.

radio-browser needs to implement this on icecast or nginx, to verify that your radio is legit.
![image](https://user-images.githubusercontent.com/24932514/198502104-2d42c9ac-03c3-4e84-ab3c-b9bed0072203.png)

**Comment by iamflorencejay** [comment]
> I'll raise this up and see what we can do as a long term solution.

Also, http-headers are now implemented on 2.4.1 and above, so the main goal is probably update @ItsMitchh

**Comment by gunsar** [comment]
now icecast-kh new version kh16, very good to load play link stream with SSL faster than kh15. hopefully azuracast will update soon @ItsMitchh

**Comment by iamflorencejay** [comment]
@ItsMitchh wen pull?

**Comment by ItsMitchh** [comment]
We note that a very recent Icecast-KH version has been deployed. We'll review when we have the time to do so.

**Comment by iamflorencejay** [comment]
This is now a great update, can you check it if this can be implemented?

**Comment by gunsar** [comment]
That's right, I've tried the update from Karl, and it runs normally on Linux and Windows. maybe in the near future Karl will release stable version of kh17 https://github.com/karlheyes/icecast-kh/issues/378#issuecomment-1376014439
